Assignment2/chrisdumi-fromJSONtoJSON-LD.jsonld

```diff
@@ -0,0 +1,38 @@
+
+{
+     "@context": {
+               "vocab": "http://upmsemanticweb.com/students#",
+               "fullName": "vocab:fullName",
+               "hometown": "vocab:hometown",
+               "age": "vocab:age",
+               "almaMater": "vocab:almaMater",
+               "parents": "vocab:parents",
+               "subjects": "vocab:subjects",
+               "name": "subjects:name",
+               "teachers": "subjects:teachers",
+               "Description": "subjects:Description"

```

Comment:
These subject attribute definitions work, but semantically they aren't ok. The best way to define them might be defining a new prefix for the subject concept and then use it to define their attributes.
You don't need to change them, but remember to provide good definitions for the future.